src/transformers/models/bigbird_pegasus/modeling_bigbird_pegasus.py

```diff
@@ -25,6 +25,8 @@
 from torch import nn
 from torch.nn import BCEWithLogitsLoss, CrossEntropyLoss, MSELoss
 
+from transformers.pytorch_utils import torch_int_div
```

Comment:
Please use relative imports, like for everything else (here and in all other modules).